HelpSource/Classes/Server.schelp

```diff
@@ -39,29 +39,29 @@ argument:: name
 a symbol;  each Server object is stored in one global classvariable under its name.
 
 argument:: addr
-an optional instance of link::Classes/NetAddr::, providing host and port.
+an optional instance of the link::Classes/NetAddr::, providing host and port.
```

Comment:
we usually say "an instance of `NetAddr`" and not: "an instance of the  `NetAddr`".